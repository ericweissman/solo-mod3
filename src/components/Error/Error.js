import React from 'react'

const Error = (props) => {
  return (
    <div className='error'>
      <h4>Dadgummit!</h4>
      <p>Looks like the computer is on the fritz. Says here it's because:</p> 
      <p className='error-message'>{props.message}!</p>
    </div>
  )
}

export default Error