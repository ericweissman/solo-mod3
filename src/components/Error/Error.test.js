import React from 'react'
import { shallow } from 'enzyme'
import Error from '../Error/Error'

describe('Error', () => {
  let wrapper
  const mockMessage = '404'
  it('should match the correct snapshot', () => {
    wrapper = shallow(<Error message={mockMessage}/>)
    expect(wrapper).toMatchSnapshot()
  })
})