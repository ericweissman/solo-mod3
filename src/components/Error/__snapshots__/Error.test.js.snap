// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should match the correct snapshot 1`] = `
<div
  className="error"
>
  <h4>
    Dadgummit!
  </h4>
  <p>
    Looks like the computer is on the fritz. Says here it's because:
  </p>
  <p
    className="error-message"
  >
    404
    !
  </p>
</div>
`;
